packages/timeline-state-resolver/src/integrations/vizMSE/$schemas/actions.json

```diff
@@ -1,6 +1,24 @@
 {
 	"$schema": "../../../$schemas/action-schema.json",
 	"actions": [
+		{
+			"id": "vizReset",
+			"name": "Reset",
+			"description": "Purges all rundowns, clears the engine and reactivates the Viz playlist",
+			"destructive": true,
+			"timeout": 10000,
+			"payload": {
+				"type": "object",
+				"properties": {
+					"activeRundownPlaylistId": {
+						"name": "Rundown playlist ID",
+						"description": "Optional property",
```

Comment:
Should this be better described maybe? Is this the Rundown Playlist ID in Sofie, or some internal Viz ID?